students/JoeNunnelley/session03/list_lab.py

```diff
@@ -0,0 +1,157 @@
+    print()
+    fruit = ['Apples', 'Pears', 'Orange', 'Peaches']
+    print(fruit)
+    response = input('Add fruit > ')
+    fruit.append(response)
+    print(fruit)
+    response = input('Get fruit number >')
+    try:
+        print('{} => {}'.format(response, fruit[int(response)-1]))
+    except IndexError as identifier:
+        print("Index not found in list: {}\n{}".format(int(response) - 1, identifier))
+
+    fruit = ["Lychee"] + fruit
+    print(fruit)
+    fruit.insert(0, "Guava")
+    print(fruit)
+
+    for _ in fruit:
+        if "P" in str.upper(_):
+            print(_)
+
+    return fruit
+
+
+def series2(fruits_in):
+    """
+    Series 2
+
+    Using the list created in series 1 above:
+
+        Display the list.
+        Remove the last fruit from the list.
+        Display the list.
+        Ask the user for a fruit to delete, find it and delete it.
+        (Bonus: Multiply the list times two. Keep asking until a match is found.
+        Once found, delete all occurrences.)
+    """
+    print()
+    print('>> Series 2 <<')
+    print()
+    print(fruits_in)
+    fruits_in.pop()
+    print(fruits_in)
+    response = input('Which fruit to delete >')
+
+    try:
+        fruits_in.remove(response)
+    except ValueError as identifier:
+        print("Did not find this item: {}\n{}".format(response, identifier))
+
+    print(fruits_in)
+    fruits_in = fruits_in * 2
+    print(fruits_in)
+    match = False
+
+    while not match:
+        print(fruits_in)
+        response = input('What fruit to eat? >')
+        if response in fruits_in:
+            while response in fruits_in:
+                try:
+                    fruits_in.remove(response)
+                    print("Chomp! {} ate {}".format(fruits_in, response))
+                except ValueError as identifier:
+                    print("Did not find this item: {}\n{}".format(response, identifier))
+
+            match = True
+
+
+def series3(fruits_in):
+    """
+    Series 3
+
+    Again, using the list from series 1:
+
+        Ask the user for input displaying a line like “Do you like apples?” for
+        each fruit in the list (making the fruit all lowercase).
+        For each “no”, delete that fruit from the list.
+        For any answer that is not “yes” or “no”, prompt the user to answer with
+        one of those two values (a while loop is good here)
+        Display the list.
+    """
+    print()
+    print('>> Series 3 <<')
+    print()
+    fruits_local = fruits_in[:]
+    for fruit in fruits_in:
+        response = None
+        while response not in ['yes', 'no']:
+            print(fruits_local)
+            response = input("Do you like {}? (yes|no)".format(str.lower(fruit)))
+
+            if response == "no":
+                print('Throwout {}'.format(fruit))
+                while fruit in fruits_local:
+                    try:
+                        fruits_local.remove(fruit)
+                    except ValueError as identifier:
```

Comment:
and why "identifier?" careful with your naming - what you are getting is an Exception object.